app/components/offering-editor-learnergroups.js

```diff
@@ -0,0 +1,47 @@
+import Ember from 'ember';
+
+const { Component, computed, isEmpty, isPresent } = Ember;
+const { sort, filter } = computed;
+
+export default Component.extend({
+  classNames: ['offering-editor-learnergroups'],
+
+  revisedLearnerGroups: computed('learnerGroups', {
+    get() {
```

Comment:
It's the latest `getter` for CPs. You can also have a `set` but we are not explicitly setting the CP so not necessary. `readonly()` will block all attempts to set it.

http://emberjs.com/api/classes/Ember.ComputedProperty.html